index.bs

```diff
@@ -517,18 +517,14 @@ type: dfn
 
   <h3 id="strict-checking"><code>Strict Mixed Content Checking</code></h3>
 
-  This specification renders the [[mixed-content#strict-checking]] mode and
-  the <code>block-all-mixed-content</code> CSP directive obsolete,
+  This specification obsoletes some parts of earlier versions of the specification —
+  including the <code>block-all-mixed-content</code> CSP directive,
```

Comment:
```suggestion
  An earlier version of this specification defined the <code>block-all-mixed-content</code> CSP directive. It is now obsolete,
```

And maybe this whole section should be non-normative since we don't need to normatively undefine things that no current specification defines?